201-sql-advanced-threat-protection-server-policy-optional-db/README.md

```diff
@@ -0,0 +1,14 @@
+# Deploy an Azure SQL Server with Advanced Threat Protection enabled and an optional Azure SQL Database
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-sql-advanced-threat-protection-server-policy-optional-db%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-sql-advanced-threat-protection-server-policy-optional-db%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
```

Comment:
I would add a link to: https://docs.microsoft.com/en-us/azure/sql-database/sql-advanced-threat-protection 

and show the final version to Ronit.

Reply:
The link is already there :) (look at the end of the file - "for more information...")